### ISSUE TITLE
Install ERROR:root:(<type 'exceptions.ImportError'>, ImportError('cannot import name _FunctionGenerator',),

### ISSUE THREAD

**Issue by codyli2002hk** [domogik/domogik#300]
During install on ubuntu 14.04.1, the following error happened
mysql version : 5.5.47
Thank you for your help.

Key log_level [error]: 
  ==> Writing the config file  
SYS.PATH=['/opt/dmg/domogik-0.4.2', '/opt/dmg/domogik-mq-1.3/src', '/usr/local/lib/python2.7/dist-packages/pyzmq-15.2.0-py2.7-linux-x86_64.egg', '/usr/local/lib/python2.7/dist-packages/simplejson-3.8.2-py2.7-linux-x86_64.egg', '/usr/lib/python2.7/dist-packages', '/usr/lib/python2.7', '/usr/lib/python2.7/plat-x86_64-linux-gnu', '/usr/lib/python2.7/lib-tk', '/usr/lib/python2.7/lib-old', '/usr/lib/python2.7/lib-dynload', '/usr/local/lib/python2.7/dist-packages/ephem-3.7.6.0-py2.7-linux-x86_64.egg', '/usr/local/lib/python2.7/dist-packages/python_magic-0.4.11-py2.7.egg', '/usr/local/lib/python2.7/dist-packages/tornado-4.3-py2.7-linux-x86_64.egg', '/usr/local/lib/python2.7/dist-packages/PyMySQL-0.7.2-py2.7.egg', '/usr/local/lib/python2.7/dist-packages/backports_abc-0.4-py2.7.egg', '/usr/local/lib/python2.7/dist-packages/certifi-2016.02.28-py2.7.egg', '/usr/local/lib/python2.7/dist-packages/singledispatch-3.4.0.3-py2.7.egg', '/usr/local/lib/python2.7/dist-packages/backports.ssl_match_hostname-3.5.0.1-py2.7.egg', '/usr/local/lib/python2.7/dist-packages', '/usr/lib/python2.7/dist-packages/PILcompat', '/usr/lib/python2.7/dist-packages/gtk-2.0', '/usr/lib/python2.7/dist-packages/ubuntu-sso-client', '/opt/dmg/domogik-0.4.2/src', '/opt/dmg/domogik-0.4.2/src/flask-themes2']
USER=root
========= TRACEBACK =============
Traceback (most recent call last):
  File "./install.py", line 504, in install
    dbi.create_db()
  File "/opt/dmg/domogik-0.4.2/src/domogik/install/db_install.py", line 64, in create_db
    from sqlalchemy_utils import database_exists, create_database
  File "/usr/local/lib/python2.7/dist-packages/sqlalchemy_utils/__init__.py", line 1, in <module>
    from .aggregates import aggregated  # noqa
  File "/usr/local/lib/python2.7/dist-packages/sqlalchemy_utils/aggregates.py", line 370, in <module>
    from sqlalchemy.sql.functions import _FunctionGenerator
ImportError: cannot import name _FunctionGenerator

=================================
ERROR:root:(<type 'exceptions.ImportError'>, ImportError('cannot import name _FunctionGenerator',), <traceback object at 0x7f03c4c013f8>)
 ==> (<type 'exceptions.ImportError'>, ImportError('cannot import name _FunctionGenerator',), <traceback object at 0x7f03c4c013f8>)

**Comment by Cereal2nd** [comment]
whats your version of sqlalchemy
, can be found out by runnign a "pip freeze"

**Comment by codyli2002hk** [comment]
SQLAlchemy==0.8.4
SQLAlchemy-Utils==0.32.1

**Comment by tikismoke** [comment]
Guigui as same problems in 0.4.2 today's irc log.

**Comment by GuillaumeRousseau** [comment]
That's right, I have the same versions, same problem
SQLAlchemy==0.8.4
SQLAlchemy-Utils==0.32.1

Domogik 0.4.2 on Ubuntu 14.04.3

**Comment by fritz-smh** [comment]
Could you run : 

```
apt-cache policy python-SQLAlchemy
```

And also run a python shell and do the following : 
```
# python
>>> import sqlalchemy
>>> print sqlalchemy.__file__
/usr/local/lib/python2.7/dist-packages/sqlalchemy/__init__.pyc
>>> print sqlalchemy.__version__
1.0.9
```

**Comment by fritz-smh** [comment]
For information, in develop install script (0.5), I added a line to display sql alchemy release. It may be helpful later on

**Comment by Cereal2nd** [comment]
we should add something like sqlalchemy >= 1.0

as this is something that was added only in that version

**Comment by fritz-smh** [comment]
Notice : this is only fixed for the next release 0.5

You should manually upgrade sqlalchemy on your side for 0.4.2
